lifecycle.go

```diff
@@ -132,10 +133,10 @@ func (d LifeCycle) Run(testplanName string) {
 
 		// save workerCount to consul
```

Comment:
yes. I think it'll be good to move it out. This anon func is already too long. I haven't thought of a good design to move it out yet.